src/test/scala/com/lunatech/lunabot/LunabotServletSpec.scala

```diff
@@ -1,16 +1,31 @@
 package com.lunatech.lunabot
 
+import javax.servlet.http.HttpServletRequest
+
+import org.scalatra._
+import scalate.ScalateSupport
 import org.scalatra.test.specs2._
 
+import scala.util.DynamicVariable
+
 // For more on Specs2, see http://etorreborre.github.com/specs2/guide/org.specs2.guide.QuickStart.html
 class LunabotServletSpec extends ScalatraSpec { def is =
   "GET / on LunabotServlet"                     ^
     "should return status 200"                  ! root200^
+  "POST /repl request on LunabotServlet" ^
```

Comment:
Also, the test says POST request, but you are only testing the function. It should say 'echoResp should .... " or similar.